betamax/serializers/proxy.py

```diff
@@ -27,6 +27,13 @@ def __init__(self, serializer, cassette_path, allow_serialization=False):
         self.cassette_path = cassette_path
 
     def _ensure_path_exists(self):
+        directory, _ = os.path.split(self.cassette_path)
+        if directory != '':
+            if not os.path.isdir(directory):
+                raise OSError(
```

Comment:
What do you think of `betamax.exceptions.CassetteError`? Or would you rather something more specific?